addons/mail/static/src/components/thread_view_topbar/thread_view_topbar.xml

```diff
@@ -23,11 +23,23 @@
                             Discuss
                         </div>
                     </t>
-                    <t t-if="threadViewTopBar.thread and threadViewTopBar.thread.description">
+                    <t t-if="threadViewTopBar.thread and threadViewTopBar.thread.isChannelDescriptionChangeable">
                         <div class="o_ThreadViewTopbar_threadDescriptionSeparator flex-shrink-0 mx-2"/>
-                        <div class="o_ThreadViewTopbar_threadDescription text-truncate" t-att-title="threadViewTopBar.thread.description">
-                            <t t-esc="threadViewTopBar.thread.description"/>
-                        </div>
+                        <t t-if="!threadViewTopBar.isEditingThreadDescription">
+                            <t t-if="threadViewTopBar.thread.description">
+                                <div class="o_ThreadViewTopbar_threadDescription text-truncate px-1" t-att-title="threadViewTopBar.thread.description" t-on-click="threadViewTopBar.onClickTopbarThreadDescription" t-on-mouseenter="threadViewTopBar.onMouseEnterTopBarThreadDescription" t-on-mouseleave="threadViewTopBar.onMouseLeaveTopBarThreadDescription" t-att-class="{ 'o-isMouseOverThreadDescription': threadViewTopBar.isMouseOverThreadDescription }">
+                                    <t t-esc="threadViewTopBar.thread.description"/>
+                                </div>
+                            </t>
+                            <t t-if="!threadViewTopBar.thread.description and threadViewTopBar.thread.isChannelDescriptionChangeable">
+                                <div class="o_ThreadViewTopbar_threadAddDescriptionEmptyLabel text-truncate" t-att-title="ADD_DESCRIPTION_LABEL" t-on-click="threadViewTopBar.onClickTopbarThreadDescription">
+                                    <t t-esc="ADD_DESCRIPTION_LABEL"/>
```

Comment:
No need for a variable in that case, text can be in the template and it will be translated correctly.

It's only necessary for title tags that are given as props (which are string) to other components.

Also having title to put the same value as the node seems useless.